app/libs/ReimbursementProvider.tsx

```diff
@@ -45,6 +45,8 @@ export const ReimbursementProvider = ({ children }: PropsWithChildren) => {
   useEffect(() => {
     if (isConfirmed && cast && hash) {
       storeReimbursment({ castHash: cast.hash, transactionHash: hash }).then(setReimburments);
+      // TODO: reply to cast that, reimbursement is paid
+      respondToReimbursementCast(cast.hash);
```

Comment:
Can we pass the hash in here and construct a link to the reimbursement tx on the base block explorer and add that link in to the contents of the reply cast? So:
respondToReimbursementCast(cast.hash, hash);

*comment conitinues on the function where the reimbursement tx hash is passed